### PR TITLE
This PR tests the contents of showtime up to the Milestone 2 contents

### PR DESCRIPTION
**Pull request by Gutogmaia**
This PR does not add anything new, as it just tests everything I have done up until now in the LectureScreen file.
In this version, 7 tests pass with line coverage of 84.05% in LectureScreen tests.

I was able to test:

- Pdf display
- base view of the screen
- rotation changes
- audiotranscript display
- questions creation and display.

All of this for the features in showtime lectureScreen

### CODE REVIEW

**Review by Gwen678 — APPROVED** [review]
This test code for the LectureScreen component is well structured and well designed. The use of mocks for Firebase dependencies, permissions management, and native components is consistent and allows to simulate different scenarios without impacting the tests. The tests cover essential functionality, such as navigating between PDF pages, rendering the default audio transcript, and navigating via icon buttons. The code is clean and mockReturnValues ​​are used effectively to ensure that external dependencies do not disrupt the tests. Great job!

File: edweiss-app/__test__/showtime/LectureScreen.test.tsx [hunk 1]
```diff
@@ -1,180 +1,277 @@
...
+        }
+    };
+});
+
+// Mock for @expo/vector-icons to render a div with accessibility label
+jest.mock('@expo/vector-icons', () => {
+    const React = require('react');
+    return {
+        Ionicons: ({ name, onPress, ...props }: { name: string, onPress?: () => void }) => (
+            <div {...props} onClick={onPress} data-testid={name} aria-label={name} />
```
Comment (FerdinandGomerChiu) [inline edweiss-app/__test__/showtime/LectureScreen.test.tsx:200]: Why did you use a div element? This seems strange, as we are in React-native.

Comment (Gutogmaia) [reply edweiss-app/__test__/showtime/LectureScreen.test.tsx:200]: Yes you are right, this could have been a TView instead to maintain consistency thanks

**Review by FerdinandGomerChiu — APPROVED** [review]
I see you used the wiki for some of the mocking, this is great work!

**Pull request merged**